src/main/java/com/deavensoft/paketomat/city/CityController.java

```diff
@@ -0,0 +1,129 @@
+package com.deavensoft.paketomat.city;
+
+import com.deavensoft.paketomat.center.model.City;
+import com.deavensoft.paketomat.city.dto.CitiesDto;
+import com.deavensoft.paketomat.city.dto.CityDto;
+import com.deavensoft.paketomat.mapper.CitiesMapper;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@RestController
+@Slf4j
+@RequestMapping("api/cities")
+
+public class CityController {
+    @Autowired
+    private final CityService cityServiceImpl;
+    @Value("${spring.url}")
+    private   String url;
+    @Value("${spring.header.host.name}")
```

Comment:
Keep in mind, that this is not a Spring configuration parameter, it's the configuration for 3rd part API. 
For example, the key should look like _external.api.city.header.host.name_.

- [ ] Rename keys in the configuration file for City API

Reply:
Done.